src/lib.rs

```diff
@@ -293,7 +293,7 @@ mod tests {
             .open(path.clone())
         {
             Ok(mut file) => {
-                file.write_all(&format!("{}", thread_rng().gen::<i32>()).into_bytes()[..])
+                file.write_all(&format!("{}", thread_rng().r#gen::<i32>()).into_bytes()[..])
```

Comment:
What's the affect of this change here? The only reference I see to this is regarding string literals?

Reply:
> use rand::{thread_rng, Rng};
Is this third-party then? Or something std? I think a better solution would be to use a method that **isn't** now a keyword, assuming it's either something you have control over or wouldn't require massive refactoring.

Reply:
`thread_rng` is a third party crate, and only used in tests. I'd like to update `thread_rng` as a dependency, to a version that renames that function, but I wanted to do it in a separate patch.